### PR TITLE
feat: Supports monorepo

### PR DESCRIPTION
**Pull request by raytung**
PoC change for https://github.com/asbiin/semantic-release-github-pullrequest/issues/86

### CODE REVIEW

**Comment by asbiin** [comment]
@raytung thank you for this contribution! The tests are failing, would you have a look?

**Comment by raytung** [comment]
@asbiin Apologies for the late reply, I've updated the tests now :+1:

**Comment by asbiin** [comment]
@raytung I've regenerated the package-lock.json using node 16.

File: lib/publish.js [hunk 1]
```diff
@@ -27,6 +28,8 @@ module.exports = async (pluginConfig, context) => {
     baseRef,
   } = resolveConfig(pluginConfig, context);
   const { owner, repo } = parseGithubUrl(repositoryUrl);
+  const absoluteRepoPath = (await execa('git', ['rev-parse', '--show-toplevel'])).stdout;
```
Comment (asbiin) [inline lib/publish.js:31]: I know this plugin is made to work on git repos, but what if it actually not run on a git repo? This command might fail. In that case, the fallback value for `absoluteRepoPath` can be `./`. Shouldn't we do a try/catch to make it happen?

**Review by asbiin — CHANGES_REQUESTED** [review]
That's great, thank you @raytung 
I have one concern with the `git` command, see my comment below

File: lib/publish.js (hunk 1 above)
Comment (raytung) [reply lib/publish.js:31]: Sure sounds good. :+1:

Comment (raytung) [reply lib/publish.js]: Fixed up now.

**Review by asbiin — APPROVED** [review]
Sound good now, thank you!

**Pull request merged**